code/modules/clothing/clothing.dm

```diff
@@ -329,6 +329,38 @@ BLIND     // can't see anything
 	body_parts_covered = HEAD
 	slot_flags = SLOT_HEAD
 	species_restricted = list("exclude","Muton")
+		// this is for hoods
```

Comment:
This is not how you comment an entire block of code, you start it with ```/*``` and end it with ```*/```